assets/src/state.ts

```diff
@@ -48,7 +48,7 @@ export const selectRoute = (routeId: RouteId): SelectRouteAction => ({
   payload: { routeId },
 })
 
-interface DeselectRouteAction {
+export interface DeselectRouteAction {
```

Comment:
what else uses this?

Reply:
We use it to construct one type in https://github.com/mbta/skate/pull/750/files#diff-92d87afe2518867c84b521efaa759c23. That said, I find it kind of inelegant exporting just this one interface, if you have a suggestion for not having to do that I'd be interested.

Reply:
You could make it the type `Dispatch` (from `state.ts`, not React's `Dispatch` type).